exercises/isogram/isogram_test.go

```diff
@@ -2,29 +2,13 @@ package isogram
 
 import "testing"
 
-var testCases = []struct {
-	word     string
-	expected bool
-}{
-	{"duplicates", true},
-	{"eleven", false},
-	{"subdermatoglyphic", true},
-	{"Alphabet", false},
-	{"thumbscrew-japingly", true},
-	{"Hjelmqvist-Gryb-Zock-Pfund-Wax", true},
-	{"Heizölrückstoßabdämpfung", true},
-	{"the quick brown fox", false},
-	{"Emily Jung Schwartzkopf", true},
-	{"éléphant", false},
-}
-
 func TestIsIsogram(t *testing.T) {
 	for _, c := range testCases {
-		if IsIsogram(c.word) != c.expected {
-			t.Fatalf("FAIL: Word %q, expected %v, got %v", c.word, c.expected, !c.expected)
+		if IsIsogram(c.input) != c.expected {
+			t.Fatalf("FAIL: Word %q, expected %t, got %t", c.input, c.expected, !c.expected)
```

Comment:
Do you think its worth adding the test case description here as an indicator for what went wrong?

Reply:
I think it's better, added.